src/main/kotlin/uk/gov/justice/digital/hmpps/hmppspersononprobationuserapi/resource/UserResourceController.kt

```diff
@@ -0,0 +1,126 @@
+package uk.gov.justice.digital.hmpps.hmppspersononprobationuserapi.resource
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.Parameter
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import org.springframework.http.MediaType
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.hmppspersononprobationuserapi.config.ErrorResponse
+import uk.gov.justice.digital.hmpps.hmppspersononprobationuserapi.service.UserService
+
+@RestController
+@Validated
+@RequestMapping("/person-on-probation-user", produces = [MediaType.APPLICATION_JSON_VALUE, MediaType.IMAGE_JPEG_VALUE])
```

Comment:
Should these endpoint be protected behind a role?

Reply:
Checked with @tomhalson:  yes it should be protected for now we will use this role, later we will have a separate role.